applications/nucleo_io/targets/freertos.armv6m.st-stm32f091rc-nucleo-dev-board/main.cxx

```diff
@@ -474,11 +475,469 @@ constexpr const Gpio *const kPortExt0[] = {
 openlcb::MultiConfiguredPC ext0_pcs(
     stack.node(), kPortExt0, ARRAYSIZE(kPortExt0), cfg.seg().ext0_pc());
 
-#endif // if num extboards > 0
+#endif // if num mcpios = 2
+
+#if NUM_MCPIOS == 4
+
+MCP23017 exp0(&io_executor, 0, 0, 0);
```

Comment:
we don't want to repeat those lines here that were already present inside the previous ifdef.
Instead, we can use something like this
```
#if NUM_MCPIOS > 0
first two
#endif
#if NUM_MCPIOS > 2
second two
#endif
... > 4
etc.
```